arch/xtensa/src/esp32/esp32_wdt_lowerhalf.c

```diff
@@ -72,14 +75,15 @@
 
 struct esp32_wdt_lowerhalf_s
 {
-  FAR const struct watchdog_ops_s *ops;        /* Lower half operations */
-  FAR struct esp32_wdt_dev_s   *wdt;           /* esp32 watchdog driver */
-  uint32_t timeout;                            /* The current timeout */
-  enum wdt_peripherals peripheral;             /* Indicates if it is from RTC or Timer Module */
-  uint32_t lastreset;                          /* The last reset time */
-  bool     started;                            /* True: Timer has been started */
-  xcpt_t handler;                              /* User Handler */
-  void   *upper;                               /* Pointer to watchdog_upperhalf_s */
+  FAR const struct watchdog_ops_s *ops;  /* Lower half operations */
+  FAR struct esp32_wdt_dev_s *wdt;       /* esp32 watchdog driver */
+  uint32_t timeout;                      /* The current timeout */
+  enum wdt_peripherals peripheral;       /* Indicates if it is from RTC or Timer Module */
+  uint32_t lastreset;                    /* The last reset time */
+  bool     started;                      /* True: Timer has been started */
+  xcpt_t handler;                        /* User Handler */
+  void   *upper;                         /* Pointer to watchdog_upperhalf_s */
+  spinlock_t lock;                       /* Device specific lock */
```

Comment:
nit: alignment.

Reply:
I initially haven't changed the original alignment, I just reduced the spaces.  But now I've changed it to make the members aligned as other source codes.